crates/router/src/connector/stripe/transformers.rs

```diff
@@ -622,9 +625,11 @@ impl TryFrom<enums::PaymentMethodType> for StripePaymentMethodType {
             | enums::PaymentMethodType::FamilyMart
             | enums::PaymentMethodType::Seicomart
             | enums::PaymentMethodType::PayEasy
-            | enums::PaymentMethodType::Walley => Err(errors::ConnectorError::NotImplemented(
-                connector_util::get_unimplemented_payment_method_error_message("stripe"),
-            )
+            | enums::PaymentMethodType::Walley => Err(errors::ConnectorError::NotSupported {
+                message: connector_util::get_unsupported_payment_method_error_message(),
+                connector: "stripe",
+                payment_experience: api_models::enums::PaymentExperience::RedirectToUrl.to_string(),
```

Comment:
What is the significance of `payment_experience` field in NotSupported error?

I discussed this with @Narayanbhat166 `payment_experience` can be removed from the NotSupported error